modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -216,6 +220,14 @@ export interface RecoverParams {
   walletPassphrase?: string;
 }
 
+export interface UTXOBlockExplorer {
+  baseUrl: string;
+  getAddressUnspent(string): Bluebird<UnspentInfo[]>;
+  //getTxInfo: (string)=>string; // get the full copy of tx with an id
+  // getAddressInfo(addressBase58: string): AddrInfo;
+  //decodeTx: (string)=>string;
```

Comment:
I think you missed removing dead comments here ^

Reply:
Those are for further reference, but they should have an intro line on top saying that.